src/systems/ec/implementations/vanilla/packages/ec-component-accordion/ec-component-accordion2.scss

```diff
@@ -0,0 +1,97 @@
+/**
+ * Accordion
+ * @define accordion2; weak
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+// Check if overridden dependencies are already loaded, if needed
+@include check-imports(('ec-component-button'));
```

Comment:
we should check ec-component-icon too (same for eu)

Reply:
fixed